scripts/language/english.lua

```diff
@@ -1400,4 +1400,16 @@ return {
 
 	-- yotb_sewer.lua
 	yotb_sewer = "Will finish sewing in: %s",
-}
\ No newline at end of file
+
+	-- inspectacleshat.lua
+	inspectacleshat = {
+		ready_to_use = "Ready to inspect",
+		cooldown = "Cooldown: %s",
```

Comment:
The cooldown in all the **inspectableshat** (for all languages) can be removed to reference the generic cooldown